.github/workflows/release.yml

```diff
@@ -95,6 +95,14 @@ jobs:
           npm install
           npm run docusaurus docs:version "${MAJOR_MINOR}"
 
+      - name: Stamp version into docs
+        shell: bash
+        run: |
+          version=${{ inputs.version }}
+          version=${version#v}  # Remove the 'v'
+          version=${version%.*} # Remove the last two characters
+          sed -i "s/@@CONSTELLATION_VERSION@@/${version}/g" docs/versioned_docs/version-${version}/workflows/terraform-module.md
+
```

Comment:
I don't know if we should create such a hard dependency on having `@@CONSTELLATION_VERSION@@` in the docs, which are touched quite regularly. IMO, we should go with having a generic example in the docs and let the user replace the version, as we do it in many other places. For Thomas saying it is easily overlooked if used in multiple places, which I agree with, we could make it a bash script and use a variable to interpolate the values correctly, while only having the user replace it in the variable declaration.

Reply:
is fixed through the local version variable.